packages/lib/README.md

```diff
@@ -0,0 +1,65 @@
+### `lib` Directory Readme
+
+#### Overview
+
+The `lib` directory is a new addition to our project structure, designed to
+house shared code used across our two main projects. This centralization of
+common components and utilities aims to improve code reusability and
+maintainability, ensuring that updates and bug fixes can be applied in a single
+location.
+
+#### Structure
+
+The `lib` directory contains various modules and functions that are shared
+between the projects. It is organized as follows:
+
+```
+lib/
+├── module1/
+│   ├── file1.ts
+│   └── file2.ts
+├── module2/
+│   └── file3.tsx
+└── README.md
+```
```

Comment:
Imo this could be deleted because it doesn't offer any new insights. Also, slightly incorrect because there is a `src/` directory. If you want to keep this src should be added!
```suggestion
```